### ISSUE TITLE
loki connection issue

### ISSUE THREAD

**Issue by iydoowii** [grafana/grafana-openai-monitoring#17]
Hi

Using grafana-openai-monitoring python 0.0.8 release to monitor openai on grafana cloud. 
Installed integration. Created a new token. 
App is running on google cloud run. I can see the prometheus metric on grafana cloud but app fails while sending logs to loki.

` Error sending Logs: 400 Client Error: Bad Request for url: https://logs-prod-017.grafana.net/loki/api/v1/push`

I simply copied, pasted grafana cloud openai integration detail in  section 5 - Select Monitor For Completions
so log url and user is correct. I also verified both on grafana cloud loki settings.

API token has permission to write logs.  What could be the issue?

**Comment by iydoowii** [comment]
we customized this module locally and added some debugs. It seems that our  `prompt` label value is too big. Loki default is 2048

```
# Maximum length accepted for label value. This setting also applies to the
# metric name.
# CLI flag: -validation.max-length-label-value
[max_label_value_length: <int> | default = 2048]
```
We trimmed the value of this metric to fix it to 1000. Now it is much better. But still getting errors time to time.  Contacted with grafana support team. They said logs are not receiving their side.

Also, if we got an error while writing to loki, whole chat request to fail.  Ideally, I prefer to avoid exception with an error. Just reporting to the user would be better.

In summary, the following changes would be good;
1. limit max value of labels to loki default.
2. don't give an error on writing to prometheus or loki to unblock chat request. or it can be parametric so anyone can set warning or error on failure.
3. Add a debug option to show values(logs, metrics) sent to loki/prometheus and http response details.

**Comment by iydoowii** [comment]
Also, we should be able to use custom job name to distinguish staging and prod  env metrics.
Right now, grafana cloud has single openai integration and single job(`integration/openai`).
